codegen/java/src/test/kotlin/io/spine/protodata/codegen/java/JavaCodegenSpec.kt

```diff
@@ -45,16 +45,16 @@ import org.junit.jupiter.api.Test
 @DisplayName("Java-related AST extensions should")
 internal class JavaCodegenSpec {
 
-    @Nested
-    inner class `Obtain Java class name from` {
+    @Nested inner class
+    `Obtain Java class name from` {
 
         @Test
         fun `top-level message`() {
             val typeName = "Anvil"
             val type = messageType(typeName)
             val file = protoMultipleFiles()
```

Comment:
Here and below, `file` variables should probably be `header`.

And maybe the method `protoMultipleFiles()` should be renamed as well.